client/tests/integration/smartcontracts/executor_with_custom_token/src/lib.rs

```diff
@@ -170,6 +170,7 @@ impl Executor {
     }
 }
 
+// TODO: why it warns about unused function?
```

Comment:
please remove and open a ticket if needed

Reply:
please also remove `request_with_filter_and_pagination_and_sorting`

Reply:
Actually me and @ilchu have already found the reason on his PR: https://github.com/hyperledger/iroha/pull/4049#discussion_r1391462122

Reply:
But I'll leave a comment